src/world/GameCata/Storage/DBCStructures.h

```diff
@@ -23,23 +23,7 @@ struct WMOAreaTableTripple
     int32_t adtId;
 };
 
-enum Powers
-{
-    POWER_MANA                          = 0,
-    POWER_RAGE                          = 1,
-    POWER_FOCUS                         = 2,
-    POWER_ENERGY                        = 3,
-    //POWER_HAPPINESS                     = 4,  unused 4.x.x
-    POWER_RUNE                          = 5,
-    POWER_RUNIC_POWER                   = 6,
-    POWER_SOUL_SHARDS                   = 7,
-    POWER_ECLIPSE                       = 8,
-    POWER_HOLY_POWER                    = 9,
-    POWER_ALTERNATIVE                   = 10,
-    MAX_POWERS                          = 11,
-    POWER_HEALTH                        = 0xFFFFFFFE    // (-2 as signed value)
-};
-
+///\ These will be verified and ported to Spell/Definitions/SpellEffectTarget.h when spell targeting is being rewritten -Appled
```

Comment:
Do you remember that you want to verify the "enum Power" in a few weeks? If not, you could comment the enum instead of deleting it.